lib/api/graphql/query/surveys_query.dart

```diff
@@ -16,3 +16,24 @@ const String GET_SURVEYS_QUERY = r"""
     }
   }
 """;
+
+const String GET_SURVEY_BY_ID = r"""
+  query($surveyId: ID!){
```

Comment:
This id type should be noted in the documentation 🧠  :o_o:

Reply:
it has :D 
![image](https://user-images.githubusercontent.com/13435717/124576552-9c9d6500-de76-11eb-9b15-942fb9138e4f.png)